### PR TITLE
add_get_interfaces_ip

### PR DESCRIPTION
**Pull request by chrisbnorman**
@samerd does this work?

### CODE REVIEW

**Comment by samerd** [comment]
@chrisbnorman 
seems good,
However, you need to rebase your branch and resolve the conflict

**Comment by samerd** [comment]
still showing conflict.
please use Resolve conflicts button to fix

**Comment by chrisbnorman** [comment]
ok i think were good now

File: test/unit/test_getters.py [hunk 1]
```diff
@@ -73,7 +73,7 @@ def test_get_interfaces_ip(self, test_case):
         assert len(get_interfaces_ip) > 0
         return get_interfaces_ip
         
-    @wrap_test_cases
+    @wrap_test_case
```
Comment (samerd) [inline test/unit/test_getters.py]: should be `@wrap_test_cases`

**Review by samerd — CHANGES_REQUESTED** [review]

**Comment by samerd** [comment]
sorry it is still failing. missing no_cli_session_paging_enable.txt
I will try to fix this issue later (if you do not want to wait for my fix, you can add a dummy file)

**Comment by chrisbnorman** [comment]
ok i got it a bit further, still failing. let me know what else needs to be done.

File: napalm_onyx/onyx_ssh.py [hunk 2]
```diff
@@ -705,7 +705,72 @@ def __get_ntp_stats(self):
...
+            ]
+        },
+        """
+        no_paging_enable_command = 'no cli session paging enable'
+        self.device.send_command(no_paging_enable_command)
+        command = 'show ip interface vrf all | json-print'
+        output = self.device.send_command(command)
+        if not output:
+            return {}
+
```
Comment (samerd) [inline napalm_onyx/onyx_ssh.py]: output = json.loads(output)

**Review by samerd — CHANGES_REQUESTED** [review]
your function should return a json not a string

**Comment by samerd** [comment]
@chrisbnorman 
I am sorry for the ping-pong.
But the command you are running will display the IP address of the management interfaces. we need also ethernet/loopback and vlan interfaces as well.
besides: the format of the output should be:
`{
<interface_name>: {
    ipv4 : {
         <address>: <prefix_length>
    }
}
}`

File: napalm_onyx/onyx_ssh.py (hunk 2 above)
Comment (chrisbnorman) [reply napalm_onyx/onyx_ssh.py]: i updated the above and tested which still functioning. please review

**Comment by chrisbnorman** [comment]
> @chrisbnorman
> I am sorry for the ping-pong.
> But the command you are running will display the IP address of the management interfaces. we need also ethernet/loopback and vlan interfaces as well.
> besides: the format of the output should be:
> `{ <interface_name>: { ipv4 : { <address>: <prefix_length> } } }`

ill add the other interfaces, i had manually removed them as they had internal IPs, ill randomise them

**Pull request closed without merging**